Makefile.work

```diff
@@ -283,8 +283,8 @@ reset :
 	        fi
 	        git clean -xfdf;
 	        git reset --hard;
-	        git submodule foreach --recursive git clean -xfdf;
-	        git submodule foreach --recursive git reset --hard;
+	        git submodule foreach --recursive 'git clean -xfdf || true';
+	        git submodule foreach --recursive 'git reset --hard || true';
```

Comment:
https://git-scm.com/docs/git-submodule

it suggests to use || :

Reply:
That's just different syntax which has the same result. `:` is a Bash builtin command which simply expands any arguments passed to it and returns 0 (i.e., true). I think `|| true` is more explicit and understandable.

See: https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Bourne-Shell-Builtins